extensions/wikia/AdEngine/js/video/player/ui/videoInterface.js

```diff
@@ -0,0 +1,22 @@
+/*global define, require*/
+define('ext.wikia.adEngine.video.player.ui.videoInterface', [
+	'wikia.log'
+], function (log) {
+	'use strict';
+
+	var logGroup = 'ext.wikia.adEngine.video.player.ui.videoInterface';
+
+	function setup(video, uiElements, params) {
+		uiElements.forEach(function (element) {
+			var module = 'ext.wikia.adEngine.video.player.ui.' + element;
```

Comment:
Seems like a deeper philosophical dilemma ;) Let's then keep it and see how many times we'll get an error from `require()`.